### PR TITLE
Feature/hub nav improvement

### PR DESCRIPTION
**Pull request by s6thgehr**

### CODE REVIEW

File: packages/app/src/hub/HubApp.tsx [hunk 1]
```diff
@@ -100,7 +111,7 @@ const _HubApp: ForwardRefRenderFunction<
                 myTree.metadata.type.translucent ? " saturate-0 opacity-50" : ""
               }`}
               onClick={() => {
-                updateShowHubNav(true);
+                updateShowHubNav((prev) => !prev);
```
Comment (ronnyhaase) [inline packages/app/src/hub/HubApp.tsx:114]: !showHub

File: packages/app/src/hub/HubApp.tsx [hunk 2]
```diff
@@ -36,6 +36,17 @@ const _HubApp: ForwardRefRenderFunction<
...
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      if (showHub) {
+        updateShowHubNav(true);
+      }
+    }, 5000);
+    return () => {
+      clearInterval(intervalId);
+    };
+  }, [showHub]);
```
Comment (ronnyhaase) [inline packages/app/src/hub/HubApp.tsx:48]: No good idea, whenever showHub changes you're creating an interval.
I suggest a one-time effect with a timeout, but there needs to be a way to cancel the timeout once the user clicks

**Review by ronnyhaase — CHANGES_REQUESTED** [review]

File: packages/app/src/hub/HubApp.tsx (hunk 1 above)
Comment (s6thgehr) [reply packages/app/src/hub/HubApp.tsx]: Done

File: packages/app/src/hub/HubApp.tsx (hunk 2 above)
Comment (s6thgehr) [reply packages/app/src/hub/HubApp.tsx]: I tried, but I couldn't figure that out, so I just deleted the timeout.

Comment (ronnyhaase) [reply packages/app/src/hub/HubApp.tsx]: OK, I'll take a look and split the task in the issue. Thanks!

**Review by ronnyhaase — APPROVED** [review]

**Pull request merged**